### DISCUSSION TITLE
Example for nginx

### DISCUSSION THREAD

**Discussion by bytegetter** [ggogel/seafile-containerized#32] [Q&A] [answered]
Hi,

is there a tutorial for unexprienced docker users with seafile and nginx?

Best Regards,
Bytegetter

**Comment by ggogel** [comment, marked as answer]
Hi,

what specifically do you want to achieve with nginx? Do you mean normal `nginx` or `jwilder/nginx-proxy`? If you use the recommended setup with `seafile-caddy` you simply need to forward all traffic that has your seafile domain as target to that container. Since this is most basic reverse proxy functionality you should be able to find plenty of tutorials on the web for all kind of web-servers or reverse proxy tools.
